### PR TITLE
www-apps/radicale: Version bump to 2.1.9 and 2.1.10 and bump required libs

### PR DESCRIPTION
**Pull request by henning-schild**
Signed-off-by: Henning Schild <henning@hennsch.de>

### CODE REVIEW

**Comment by henning-schild** [comment]
was #8285

File: www-apps/radicale/radicale-2.1.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit distutils-r1 eutils user
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="http://radicale.org/"
```
Comment (Hawk777) [inline www-apps/radicale/radicale-2.1.9.ebuild]: The Radicale homepage is available over HTTPS, and even appears to redirect to it. Why add a security hole here?

File: www-apps/radicale/radicale-2.1.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+		eerror "If you have kept the Gentoo-default database configuration, this will work:"
+		eerror "1. Stop any running instance of Radicale."
+		eerror "2. Run \`radicale --export-storage ~/radicale-exported\`."
+		eerror "3. Run \`chown -R radicale: ~/radicale-exported\`"
+		eerror "4. Run \`mv \"${RDIR}\" \"${RDIR}.old\"\`."
+		eerror "5. Install Radicale version 2."
+		eerror "6. Run \`mv ~/radicale-exported \"${RDIR}/collections\"\`."
+		eerror
+		eerror "For more details, or if you are have a more complex configuration,"
+		eerror "please see the migration guide: http://radicale.org/1to2/"
```
Comment (Hawk777) [inline www-apps/radicale/radicale-2.1.9.ebuild]: … and also here.

File: www-apps/radicale/radicale-2.1.9.ebuild (hunk 1 above)
Comment (Hawk777) [reply www-apps/radicale/radicale-2.1.9.ebuild]: (To be clear, the old ebuild had HTTPS here)

Comment (henning-schild) [reply www-apps/radicale/radicale-2.1.9.ebuild]: done, that was something the old ebuild received inbetween 3183fa37dfdabc8ccfd591ecad68e7a841e4299d

**Comment by henning-schild** [comment]
ping @mgorny

File: www-apps/radicale/radicale-2.1.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+inherit distutils-r1 eutils user
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="https://radicale.org/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (mgorny) [inline www-apps/radicale/radicale-2.1.9.ebuild]:
```
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
```

That's GPL-3+.

Comment (henning-schild) [reply www-apps/radicale/radicale-2.1.9.ebuild]: Not so sure ...
```$ md5sum COPYING /usr/portage/licenses/GPL-3
d32239bcb673463ab874e80d47fae504  COPYING
d32239bcb673463ab874e80d47fae504  /usr/portage/licenses/GPL-3```

Comment (henning-schild) [reply www-apps/radicale/radicale-2.1.9.ebuild]: I am not sure i get that so please tell me exactly how to fix that. Do i set "GPL-3+" instead or do i go for an and/or list. And would that change also affect the other ebuilds, so i would have a first commit changing the license of them all and bumping to -r+1 followed by the bump ebuild?

Comment (mgorny) [reply www-apps/radicale/radicale-2.1.9.ebuild]: Just change it to GPL-3+ and you're done. No need to bump for a minor change like that (the original license was narrower, so there's no danger associated with keeping the old version).

**Comment by mgorny** [comment]
Thanks. Let's give @maksbotan some more time to review it. Please ping me if he doesn't respond in a while.

**Comment by mgorny** [comment]
Ok, I think we can assume maintainer timeout by now. I'll test it, and push everything but 2.1.9 bump. I suppose there's no need to add an old version.

**Pull request closed without merging**